csrc/preseg_passes/mark_aliases_prepare.h

```diff
@@ -19,7 +19,7 @@ class MarkAliasesPreparePass : public OptimizationPass<MarkAliasesPreparePass> {
 
  protected:
   static void runPass(Fusion* fusion);
-  static std::string name() {
+  static constexpr const char* name() {
```

Comment:
Nit: Consider `constexpr std::string_view` to be more type safe. There are too many ways to mess up a raw char pointer.